src/components/FooterContainer/Footer/Footer.js

```diff
@@ -77,23 +77,31 @@ const Footer = props => (
           </div>
         )}
 
-        <div className={style.staticLinksWrapper}>
-          {links.map(link => (
-            <NavLink
-              key={link.href}
-              to={link.href}
-              className={style.staticLink}
-            >{link.text}</NavLink>
-          ))}
-        </div>
+        <div className={style.footerLinks}>
+          <div className={style.footerLinksRow}>
+            {links.map(link => (
+              <NavLink
+                key={link.href}
+                to={link.href}
+                className={style.staticLink}
+              >{link.text}</NavLink>
+            ))}
+          </div>
 
-        <div className={style.contact}>
-          {STRINGS.CONTACT}
-          {': '}
-          <a
-            className={style.contactLink}
-            href={`mailto:${EMAIL_ADDRESSES.CITY_PERFORMANCE}?subject=${STRINGS.CONTACT_SUBJECT}`}
-          >{EMAIL_ADDRESSES.CITY_PERFORMANCE}</a>
+          <div className={style.footerLinksRow}>
+            {STRINGS.CONTACT}
+            {': '}
+            <a
+              href={`mailto:${EMAIL_ADDRESSES.CITY_PERFORMANCE}?subject=${STRINGS.CONTACT_SUBJECT}`}
+            >{EMAIL_ADDRESSES.CITY_PERFORMANCE}</a>
+          </div>
+
+          <div className={style.footerLinksRow}>
+            <a
+              href={LINKS.RAW_DATA}
+              target="_blank"
```

Comment:
Any link with `target="_blank"` must set `rel="noopener noreferrer"` to protect against [this vulnerability](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/) (however unlikely that is).

We've got a `<Link>` component that does this for you, but it also adds styling, so either adding `rel` or using `<Link>` is fine.